specutils/io/default_loaders/wcs_fits.py

```diff
@@ -86,6 +106,24 @@ def wcs1d_fits_loader(file_obj, spectral_axis_unit=None, flux_unit=None,
         print("Spectrum file looks like wcs1d-fits")
 
     with read_fileobj_or_hdulist(file_obj, **kwargs) as hdulist:
+        if hdu is None:
+            for ext in ('FLUX', 'SCI', 'DATA', 'PRIMARY'):
+                # For now rely on extension containing spectral data.
+                if ext in hdulist and (
+                        isinstance(hdulist[ext], (fits.ImageHDU, fits.PrimaryHDU)) and
+                        hdulist[ext].data is not None):
+                    if hdu is None or hdulist[ext] == hdulist[hdu]:
+                        hdu = ext
+                        continue
+                    else:
+                        warnings.warn(f"Found multiple data HDUs '{hdu}' and '{ext}', "
+                                      f"will read '{hdu}'! Please use `hdu=<flux_hdu>` "
+                                      "to select a specific one.", AstropyUserWarning)
```

Comment:
```suggestion
                                      f"will read '{hdu}'. Please use `hdu=<flux_hdu>` "
                                      "to select a specific extension.", AstropyUserWarning)
```
Minor quibble, would rather be specific ("extension" or "hdu") here than say "one".